src/test/java/org/apache/commons/imaging/color/ColorConversionsTest.java

```diff
@@ -39,7 +40,7 @@ public void testRGBtoCMYK() {
             Debug.debug("cmyk_cmy: " + cmyk_cmy);
             Debug.debug("cmyk_cmy_rgb: " + cmyk_cmy_rgb + " (" + Integer.toHexString(cmyk_cmy_rgb) + ")");
 
-            assertEquals((0xffffff & cmyk_cmy_rgb), (0xffffff & rgb));
+            assertEquals(toHexString(0xffffff & rgb), toHexString(0xffffff & cmyk_cmy_rgb));
```

Comment:
"expected" and "actual" values where in the wrong order. Also, comparing the hex strings is much more comprehensible in test failure messages.